### PR TITLE
Improve help texts of the diff, issue & create-keys app

### PR DESCRIPTION
**Pull request by MeBrei**
More detailed description of the diff/issue/create-keys app commands. Also in regard to the idea of using the help texts in the documentation.

### CODE REVIEW

File: bin/rad-issue [hunk 1]
```diff
@@ -22,22 +22,31 @@
...
-
-     list         - Select from issues to view or comment on
+        rad issue list " list-opts-help "
+        rad issue new
+        rad issue [show | close] <issue-number>
+        rad issue comment <issue-number> <comment>
+        rad issue help
+
+     list         - Lists all issues
+                    The list can be filtered by state:
```
Comment (jameshaydon) [inline bin/rad-issue]: Maybe extract this string to a function to avoid duplication with the rad-diff help.

Comment (MeBrei) [reply bin/rad-issue]: I would leave it for now. Maybe we could have a future PR where we also extract all non-error user feedback to make them more consistent.

**Comment by xla** [comment]
In the user flows we had `rad key create` to leave the possibilty to extend the namespace further down the road (listing and general management). Is there strong reason to not move to that?

**Comment by MeBrei** [comment]
> In the user flows we had `rad key create` to leave the possibilty to extend the namespace further down the road (listing and general management). Is there strong reason to not move to that?

@xla No strong reason against that. I will create a ticket for it and change it in a different PR as there will be more changes than only the help text.

**Pull request merged**